openapi/wicketd.json

```diff
@@ -1063,7 +1063,7 @@
         }
       },
       "Baseboard": {
-        "description": "Describes properties that should uniquely identify a Gimlet.",
+        "description": "Describes properties that should uniquely identify a system.",
```

Comment:
same here

Reply:
Given that wicket, unlike bootstrap agent, actually reports about switches and power shelf controllers, which also have the exact same underlying information (in hardware and the SP) is the wicketd api here actually only specific to compute sleds?

Reply:
I don't actually know.

However, I'm (mildly) concerned about it getting a bit wordy. I've tried to split the difference by calling it, "an Oxide system" in this latest version; I'm not sure if that threads the needle particularly usefully, but perhaps Oxide addresses the need for specificity (versus something from some other vendor) and pairing with "system" keeps it sufficiently general that it's inclusive of switches, PSCs (even though those aren't technically ours) and compute sleds.

What do you think?

Reply:
> and that also incorporates support for non-Oxide hardware!

ugh, sad :(